lab_2/main.py

```diff
@@ -46,7 +77,13 @@ def find_lcs_length(first_sentence_tokens: tuple, second_sentence_tokens: tuple,
     :param plagiarism_threshold: a threshold
     :return: a length of the longest common subsequence
     """
-    pass
+    lcs_matrix = fill_lcs_matrix(first_sentence_tokens,second_sentence_tokens)
+    if lcs_matrix == [] or not isinstance(plagiarism_threshold,float) or plagiarism_threshold < 0 \
```

Comment:
`lcs_matrix == []` is it really needed? That explicit?

Reply:
why not? in function "fill_lcs_matrix" the same explicits for type of tokens